lib/kissable/ab.rb

```diff
@@ -19,8 +19,10 @@ def initialize(test_name, groups=nil, ratios=nil)
       validate_ratios
     end
 
-    def group(cookies)
-      @cookies = cookies
+    # Assigns a test group based on the 'abid' cookie or login (email).
+    # Takes either a cookies object (Hash-like) or login (String).
+    def group(object)
+      (object.is_a? String) ? (@login = object) : (@cookies = object)
```

Comment:
So the functionality will be as such:

1. If cookies are passed in, look for a cookie, if present use. Else, create and write a cookie.
2. If a login is passed in, use the login to construct the A/B group the user falls in. NEVER USE COOKIES.

@mattysads If the above is correct, say "YES" or some other shipping icon. :)